src/row.rs

```diff
@@ -0,0 +1,113 @@
+/// A macro for creating a [`Row`] using vec! syntax.
+///
+/// `row!` is similar to the [`vec!`] macro, but it returns a [`Row`] instead of a `Vec`.
+///
+/// # Examples
```

Comment:
Add the empty case example

Reply:
I'm not sure I quite follow this comment.

Reply:
`row![]` is not documented.

Reply:
Ah, that makes sense. I've added it in the other PR.